x-pack/plugins/security_solution/server/graphql/source_status/resolvers.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { filterIndexes } from './resolvers';
+
+describe('resolvers', () => {
+  test('it should filter single index that is has an empty string', () => {
```

Comment:
Super nit: think this and the next one should have `is` removed from `is has`.

Reply:
Fixed!